packages/astro/src/@types/astro.ts

```diff
@@ -901,6 +901,36 @@ export interface AstroUserConfig {
 	 */
 	experimentalErrorOverlay?: boolean;
```

Comment:
Shouldn't this be moved into the experimental object? Or is @bholmesdev doing that in his PR?

Reply:
I can do it here!

Reply:
Fixed in [9e4a190](https://github.com/withastro/astro/pull/5297/commits/9e4a19076d9fa6e803d440f543c386e940116e57)